src/runtime/components/User.vue

```
@@ -73,7 +73,7 @@ const ui = computed(() => tv({ extend: tv(theme), ...(appConfig.ui?.user || {})
 </script>
 
 <template>
-  <Primitive :as="props.as" :data-orientation="props.orientation" data-slot="root" :class="ui.root({ class: [props.ui?.root, props.class] })" @click="props.onClick">
+  <Primitive :as="props.as" v-bind="!props.to ? $attrs : {}" :data-orientation="props.orientation" data-slot="root" :class="ui.root({ class: [props.ui?.root, props.class] })" @click="props.onClick">
```

ESLint

[error] '`@click`' should be on a new line.

(vue/max-attributes-per-line)